plugin.yaml

```diff
@@ -114,6 +114,11 @@ node_types:
                     the ssl directory.
                     If this is provided, the ssl_certificate must also be provided.
                 default: null
+            start_nagiosrest:
+                description: >
+                    Start rest service, or do it later, via EnableNagiosRest node.
```

Comment:
This should probably be changed, as it suggests that this parameter is maybe required in order to invoke EnableNagiosRest.

e.g. "Whether to start the nagiosrest service.
If this is false, it may be started later using the EnableNagiosRest operation."

Reply:
It's probably also worth noting here that while the nagiosrest service is disabled, no new monitored nodes can successfully be added, and any blueprints that specify them will fail to deploy. Also, any existing deployments that attempt to heal or scale will fail.

Reply:
This is partly true due to the latest changes in nagiosrest plugin - nodes trying to add_monitoring won't fail right away - they'll try to do so 60 times or so - which is usually enough for nagiosrest to start. I'll fix the name and the description.